### ISSUE TITLE
Apple Silicon Support

### ISSUE THREAD

**Issue by sh0ckwav** [natmlx/NatML#2]
As instructed opening an issue to track support for Apple Silicon (arm64)

`DllNotFoundException: NatML assembly:<unknown assembly> type:<unknown type> member:(null)`

**Comment by olokobayusuf** [comment]
This is coming in NatML 1.0.8.

**Comment by smtabatabaie** [comment]
I'm also facing the same problem. It works in my UnityEditor, but when I export for Mac, it doesn't work in my M1 mac, I also tried it in a non-M1 mac, and that also doesn't work and doesn't show my webcam feed. Exporting for windows also works in some systems and doesn't work on others. It would be great if there was a clear system requirement for that also. Thanks

**Comment by olokobayusuf** [comment]
Hey @smtabatabaie, support for Apple Silicon is launching in the next update. As for any issues with a camera feed, this is not a NatML issue; any camera streaming is implemented with Unity's `WebCamTexture`. You can see [NatML's current system requirements](https://docs.natml.ai/unity/prelims/requirements).

**Comment by smtabatabaie** [comment]
Thanks very much @olokobayusuf , No I don't think it's about WebCamTexture, because it works just fine in the Unity Editor when I play, it's just when I export that it shows nothing and when I traced the code gets stuck I came to this:
It goes through BlazeFaceSample.cs and tries to execute line 46 "model = modelData.Deserialize()" and in Deserialize function in MLEdgeModel.cs it tries to run line 82 "return new MLEdgeModel(session.id, graph, &flags);" and in MLEdgeModel.cs script it gets such in "NatML.CreateModel(buffer, graph.Length, reserved, out model);" and the code stops there and doesn't execute. So it's not WebCamTexture and it works fine in the Unity Editor, but it stops at line I mentioned when I export it as a Mac app and also on some Windows system. It works in some Windows system and it doesn't work in some windows systems, the systems I tried all where x64. I also couldn't run it on any of my Mac systems (M1s and Non-M1s).
Thanks again very much
